Source/View Controllers/TCCProfileViewController.swift

```diff
@@ -159,6 +182,14 @@ class TCCProfileViewController: NSViewController {
         alertWindow.beginSheetModal(for: window)
     }
 
+    func turnOnBigSurCompatibility() {
+        if model.requiresAuthorizationKey() {
+            authorizationKeySwitch.state = .on
+            // when importing, we assume they know what they are doing and don't need to see the alert
```

Comment:
This comment seems like it might be better as a function comment, or down in the `importProfile` method.

Reply:
yeah, or maybe I should leave the comment out altogether. I wanted to explain why we are not showing the alert, but not sure it is needed.